src/Microsoft.ML.StandardLearners/Standard/MultiClass/Ova.cs

```diff
@@ -38,7 +38,6 @@ namespace Microsoft.ML.Runtime.Learners
     using TDistPredictor = IDistPredictorProducing<float, float>;
     using CR = RoleMappedSchema.ColumnRole;
 
-    /// <include file='doc.xml' path='doc/members/member[@name="OVA"]' />
```

Comment:
>/// <include file='doc.xml' path='doc/members/member[@name="OVA"]' /> [](start = 4, length = 69)

Hi @sfilipi, I think you own this file? Despite its path, this documentation is very specific to ML.NET 0.1 pieplines (it is also linked in Legacy), so maybe we need something new here, or should update and re-link once we deprecate legacy?